spec/models/user_spec.rb

```diff
@@ -1,5 +1,45 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'registration validation test' do
+    it 'ensures full_name, email, password, address, cpf and birthdate presence' do
+      user = User.create
+
+      expect(user.errors[:full_name]).to include('não pode ficar em branco')
+      expect(user.errors[:email]).to include('não pode ficar em branco')
+      expect(user.errors[:password]).to include('não pode ficar em branco')
+      expect(user.errors[:address]).to include('não pode ficar em branco')
+      expect(user.errors[:cpf]).to include('não pode ficar em branco')
+      expect(user.errors[:birthdate]).to include('não pode ficar em branco')
+      expect(user.valid?).to eq(false)
+    end
+
+    it 'registrate a new user' do
```

Comment:
Acho que essa descrição não faz muito jus ao teste, o que acham de trazer mais informação?

Reply:
Ajustado como `'ensures a new user is properly created'`